src/cryptography/hazmat/backends/openssl/backend.py

```diff
@@ -56,6 +57,91 @@
                                        ["code", "lib", "func", "reason"])
 
 
+def _make_asn1_int(backend, x):
+    i = backend._lib.ASN1_INTEGER_new()
+    # i = backend._ffi.gc(i, backend._lib.ASN1_INTEGER_free)
+    backend._lib.ASN1_INTEGER_set(i, x)
+    return i
+
+
+def _make_asn1_str(backend, x, n=None):
+    if n is None:
+        n = len(x)
+    s = backend._lib.ASN1_OCTET_STRING_new()
+    # s = backend._ffi.gc(s, backend._lib.ASN1_OCTET_STRING_free)
+    backend._lib.ASN1_OCTET_STRING_set(s, x, n)
+    return s
+
+
+def _encode_name(backend, attributes):
+    resolve = {
```

Comment:
Same `OBJ_obj2txt` comment from the CSR builder. Presumably these functions are here so that the code works but whichever lands first will cause a rebase of the other :)